app/assets/stylesheets/views/_organisations.scss

```diff
@@ -20,3 +20,20 @@
     }
   }
 }
+
+.margin-bottom {
```

Comment:
This is a really vague class name. I know we don't tend to do BEM in our views, but we should probably at least scope it to be inside `.organisations` or rename it to `organisations__margin-bottom`. It feels like this is being used more as a mixin too - perhaps that would make it neater?